metron-platform/metron-hbase/metron-hbase-common/src/main/java/org/apache/metron/hbase/ColumnList.java

```diff
@@ -18,7 +18,7 @@
  *
  */
 
-package org.apache.metron.hbase.bolt.mapper;
+package org.apache.metron.hbase;
```

Comment:
I don't see `org.apache.metron.hbase.Connector` used anywhere.  Am I missing it?  If not, we should delete this too.

Reply:
Done